src/current/cockroachcloud/cloud-api.md

```diff
@@ -709,7 +844,7 @@ curl --request PUT \
 Where:
 - `{cluster_id}` is the unique ID of this cluster.
 {{site.data.alerts.callout_info}}
-The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-a-basic-cluster.md %}).
+The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-your-cluster.md %}).
```

Comment:
This is out of sync with similar text above about the cluster ID differing.

Reply:
I don't see the problem - can you please clarify?

Reply:
I don't remember.